geoh5py/data/color_map.py

```diff
@@ -85,10 +85,9 @@ def values(self, values: np.ndarray):
             self._values = np.asarray(
                 values, dtype=list(zip(self._names, self._formats, strict=False))
             )
-        self.parent.modified_attributes = "color_map"
 
         if self.parent is not None:
```

Comment:
could you get rid of the pylint disable with `if hasattr(self, "parent", None) is not None`?

Reply:
I tried, doesn't work somehow